scripts/integration_test_js.sh

```diff
@@ -118,9 +118,12 @@ else
 
     cd tests/EvmAcceptanceTests/
     npm install
-    DEBUG=true MOCHA_TIMEOUT=20000 npx hardhat test
```

Comment:
this has effect of not interleaving the hardhat output with the iso server output, it just dumps it after